src/API/api.js

```diff
@@ -49,6 +50,41 @@ export type Pool = {|
   +color_pledge: string, // hsl(240,95%,95%)
   +saturation: number,
 |};
+export type PoolDaedalusSimple = {|
+  +id: string, // hash
+  +db_ticker: ?string, // may not have a ticker
+  +db_name: ?string, // may not have a name
+  +pool_pic: ?string, // may not have a pic
+  +fullname: ?string,
+  +pledge: string,
+  +pledge_real: string, // not sure diff with "pledge"
```

Comment:
this is the pledge which is currently active with real funds, it may be more or less than the declared pledge